backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/nodes/MergedReferencesCompartmentItemNodeDescriptionProvider.java

```diff
@@ -0,0 +1,125 @@
+        super(colorProvider);
+        this.eClass = Objects.requireNonNull(eClass);
+        this.eReferences = Objects.requireNonNull(eReferences);
+        if (this.eReferences.stream().map(ETypedElement::getEType).distinct().count() != 1) {
+            throw new IllegalArgumentException();
+        }
+        this.eType = this.eReferences.stream().findAny().get().getEType();
+        this.nameGenerator = Objects.requireNonNull(nameGenerator);
+    }
+
+    @Override
+    public NodeDescription create() {
+        return this.diagramBuilderHelper.newNodeDescription()
+                .defaultHeightExpression(ViewConstants.DEFAULT_COMPARTMENT_NODE_ITEM_HEIGHT)
+                .defaultWidthExpression(ViewConstants.DEFAULT_NODE_WIDTH)
+                .domainType(SysMLMetamodelHelper.buildQualifiedName(this.eType))
+                .insideLabel(this.createInsideLabelDescription())
+                .name(this.nameGenerator.getCompartmentItemName(this.eClass, this.eReferences.get(0)))
+                .semanticCandidatesExpression(
+                        AQLConstants.AQL_SELF + ".getAllContentsByReferences(Sequence{" + this.eReferences.stream().map(ref -> "self." + ref.getName()).reduce((a, b) -> a + ',' + b).get() + "})")
+                .style(this.createCompartmentItemNodeStyle())
+                .userResizable(false)
+                .palette(this.createCompartmentItemNodePalette())
+                .synchronizationPolicy(SynchronizationPolicy.SYNCHRONIZED)
+                .build();
+    }
+
+    protected InsideLabelDescription createInsideLabelDescription() {
+        return this.diagramBuilderHelper.newInsideLabelDescription()
+                .labelExpression(AQLConstants.AQL_SELF + ".getCompartmentItemUsageLabel()")
+                .position(InsideLabelPosition.TOP_CENTER)
+                .style(this.createInsideLabelStyle())
+                .build();
+    }
+
+    protected InsideLabelStyle createInsideLabelStyle() {
+        return this.diagramBuilderHelper.newInsideLabelStyle()
+                .displayHeaderSeparator(false)
+                .labelColor(this.colorProvider.getColor(ViewConstants.DEFAULT_LABEL_COLOR))
+                .showIcon(true)
+                .withHeader(false)
+                .build();
+    }
+
+    private NodeStyleDescription createCompartmentItemNodeStyle() {
+        return this.diagramBuilderHelper.newIconLabelNodeStyleDescription()
+                .borderColor(this.colorProvider.getColor(ViewConstants.DEFAULT_BORDER_COLOR))
+                .borderRadius(0)
+                .background(this.colorProvider.getColor(ViewConstants.DEFAULT_BACKGROUND_COLOR))
+                .build();
+    }
+
+    private NodePalette createCompartmentItemNodePalette() {
+        var callDeleteService = this.viewBuilderHelper.newChangeContext()
+                .expression(AQLConstants.AQL_SELF + ".deleteFromModel()");
```

Comment:
Could you please use the new APIs from `AQLUtils` here and everywhere else in this PR?
`AQLConstants.AQL_SELF + ".deleteFromModel()"` => `AQLUtils.getSelfServiceCallExpression("deleteFromModel")`